2020/day8/solution.py

```diff
@@ -0,0 +1,75 @@
+import re
+import typing
+from collections.abc import Iterable, Callable
+
+#Type Information
+Instruction = tuple[str, int]
+
+instruction_re = re.compile(r"(?P<instruction>\w{3}) (?P<value>[+-]\d+)")
+
+class Machine:
+
+    def _acc(self, value : int) -> int:
+        self.acc += value
+        return 1
+
+    INSTRUCTION_TABLE : dict[str, Callable[['Machine',int], int]] = {
```

Comment:
```suggestion
    INSTRUCTION_TABLE : dict[str, Callable[[Machine, int], int]] = {
```
You'll need a `from __future__ import annotations` at the top. See https://www.python.org/dev/peps/pep-0563/ for details.